dipy/data/fetcher.py

```diff
@@ -117,16 +117,24 @@ def check_md5(filename, stored_md5=None):
 def _get_file_data(fname, url):
     with contextlib.closing(urlopen(url)) as opener:
         if sys.version_info[0] < 3:
-            response_size = opener.headers['content-length']
+            try:
+                response_size = opener.headers['content-length']
+            except:
+                response_size = None
         else:
             # python3.x
-            response_size = opener.getheader("Content-Length")
-
+            try:
+                response_size = opener.getheader("Content-Length")
+            except:
+                response_size = None
         with open(fname, 'wb') as data:
-            copyfileobj_withprogress(opener, data, response_size)
+            if(response_size is None):
+                copyfileobj(opener, data)
+            else:
+                copyfileobj_withprogress(opener, data, response_size)
```

Comment:
not exactly.
"data-size is approximately X MB" is printed whenever the size is defined even if 'content-length' is defined and progressbar is displayed.

I noticed some datasets in which 'content-length' and data-size both are not available.